### PR TITLE
Some small changes to docs + table and row setting

### PR DESCRIPTION
**Pull request by Norbiros**
I changed some things that were annoying me, and I added info about table and row setting.

### CODE REVIEW

File: content/docs/reference/addon-api/addon.self.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ In that case, the `reenabled` event will fire.
     <td>tag</td>
     <td><code>String</code></td>
     <td>No</td>
-    <td>The tag for filtering.</td>
+    <td>The group tag for filtering.</td>
```
Comment (mxmou) [inline content/docs/reference/addon-api/addon.self.md:146]: They're called tags in the `addon.json`.

File: content/docs/reference/addon-api/addon.self.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ In that case, the `reenabled` event will fire.
   </tr>
 </table>
 
-Gets a list of addon IDs enabled, optionally filtered using tags.
+Gets a list of addon IDs enabled, optionally filtered using group name.
```
Comment (mxmou) [inline content/docs/reference/addon-api/addon.self.md:157]: Same as above.

File: content/docs/reference/addon-manifest.md [hunk 3]
```diff
@@ -515,7 +528,7 @@ Keep in mind, few addons will be enabled by default. If you want your addon to b
 | - | - |
 | Type | `Array` |
 
-An array of additional information (e.g. warnings, notices) about the addon. Each item of the array is an object consisting of `type` (string) -either `warning` or `notice` - `text` (string) - the text to be displayed - and `id` (string) - the id of the information.
+An array of additional information (e.g. warnings, notices) about the addon. Each item of the array is an object consisting of `type` (string) -either `warning`. `notice` or `info` - `text` (string) - the text to be displayed - and `id` (string) - the id of the information.
```
Comment (mxmou) [inline content/docs/reference/addon-manifest.md:531]: This should mention that `type` is optional and defaults to info.

**Review by mxmou — CHANGES_REQUESTED** [review]

File: content/docs/reference/addon-api/addon.self.md (hunk 2 above)
Comment (Norbiros) [reply content/docs/reference/addon-api/addon.self.md:157]: But for example me when I was creating my first addon, I didn't know how I can filter, which tags, regex, something like that? But okay I can undo that change

Comment (mxmou) [reply content/docs/reference/addon-api/addon.self.md:157]: You can add a link to https://scratchaddons.com/docs/reference/addon-manifest/#tags

File: content/docs/reference/addon-manifest.md (hunk 3 above)
Comment (Norbiros) [reply content/docs/reference/addon-manifest.md]: Now it is correct?

> An array of additional information (e.g. warnings, notices) about the addon. Each item of the array is an object consisting of `id` (string) - the id of the information, `text` (string) - the text to be displayed and optional `type` (string) - either `warning`, `notice` or `info` (default type).

Comment (mxmou) [reply content/docs/reference/addon-manifest.md]: Yes, that's correct.

File: content/docs/reference/addon-api/addon.self.md (hunk 2 above)
Comment (Norbiros) [reply content/docs/reference/addon-api/addon.self.md]: Okay, so now everything is done.

**Comment by Hans5958** [comment]
@mxmou, could you give a seal of approval on this one?

File: content/docs/reference/addon-api/addon.self.md [hunk 4]
```diff
@@ -154,7 +154,7 @@ In that case, the `reenabled` event will fire.
   </tr>
 </table>
 
-Gets a list of addon IDs enabled, optionally filtered using tags.
+Gets a list of addon IDs enabled, optionally filtered using [tag](https://scratchaddons.com/docs/reference/addon-manifest/#tags).
```
Comment (mxmou) [inline content/docs/reference/addon-api/addon.self.md:157]:
```suggestion
Gets a list of addon IDs enabled, optionally filtered using a [tag](https://scratchaddons.com/docs/reference/addon-manifest/#tags).
```

File: content/docs/reference/addon-manifest.md [hunk 5]
```diff
@@ -338,11 +338,24 @@ Specify a `settings` property and provide an array of option objects.
...
 - `default` (string, required) The default value for the option. A boolean, string, or number, depending on the specified type.  
 - `min`/`max` (number, optional for `positive_integer`, `integer`, and `string` types only) For integers, the minimum/maximum value allowed, and for strings, the minimum/maximum allowed length of the value.
 - `potentialValues` (array of objects, required for `select` type only) Array of objects, with properties `id`, the value received from `addon.settings.get()`, and `name`, the user-visible option text.
 - `allowTransparency` (boolean, required for `color` type only) Whether the user should be allowed to enter transparent colors or not.
+- `row` (array of objects, only for `table` type). Every element in table contains this array of objects. Each object should contain: `name`, `id`, `type` and `default`. For example:
```
Comment (mxmou) [inline content/docs/reference/addon-manifest.md:346]: I think this should mention which types of settings a table row can contain (any setting other than another table).

**Review by mxmou — APPROVED** [review]

**Review by Hans5958 — APPROVED** [review]

**Pull request merged**